pkg/worker/coordinator.go

```diff
@@ -118,13 +183,16 @@ func (h *Handler) RouteCoordinator() {
 		"quit",
 		func(resp cws.WSPacket) (req cws.WSPacket) {
 			log.Println("Received a quit request from coordinator")
-			session, ok := h.sessions[resp.SessionID]
-			log.Println("Find ", resp.SessionID, session, ok)
+			session := h.getSession(resp.SessionID)
 
-			room := h.getRoom(session.RoomID)
-			// Defensive coding, check if the peerconnection is in room
-			if room.IsPCInRoom(session.peerconnection) {
-				h.detachPeerConn(session.peerconnection)
+			if session != nil {
```

Comment:
golang good practice is
if session, ok := h.sessions[resp.SessionID]; ok {
   // do something
} else {
   //
}

use ok to check

Reply:
the `h.getSession(resp.SessionID)` uses the same mechanism, not sure why `getSession` func was there before